src/integrated/land.jl

```diff
@@ -481,19 +481,12 @@ function soil_boundary_fluxes!(
 ) where {FT}
     bc = soil.boundary_conditions.top
     p.soil.turbulent_fluxes .= turbulent_fluxes(bc.atmos, soil, Y, p, t)
-    Soil.Runoff.update_runoff!(
-        p,
-        bc.runoff,
-        p.drivers.P_liq .+ p.snow.water_runoff .* p.snow.snow_cover_fraction .+
-        p.excess_water_flux,
-        Y,
-        t,
-        soil,
-    )
-    @. p.soil.top_bc.water =
-        p.soil.infiltration +
-        (1 - p.snow.snow_cover_fraction) *
-        p.soil.turbulent_fluxes.vapor_flux_liq
+    influx = @. p.drivers.P_liq +
```

Comment:
Is influx here different from infiltration? It seems like it is the water available at the surface that may infiltrate into the soil, and it gets separated into runoff and infiltration, but I'm not 100% sure. It would be good to add a comment clarifying

Reply:
I added comments in all 4 files with this.